### PR TITLE
Allow user to choose book order and save setting

### PR DESCRIPTION
**Pull request by jmarsican**
These changes address #142 and #1013. Also includes #1116 
- Restore menu
- Add two options to enable alphabetic order and row left to right
- Trigger navigation book sort order with menu option
- Remove action bar sort toggle button
- Implement option to fill book buttons by row (traspose) or old way

**Screenshots**

Screenshot_20210603-001718
Screenshot_20210603-001930
Screenshot_20210603-002025
tablet
Screenshot_20210603-002228

### CODE REVIEW

File: app/src/main/res/menu/choose_passage_book_menu.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/alphabetical_order_opt"
+        android:title="@string/sort_by_alphabetical"
+        android:checkable="true"/>
+
+    <item
+        android:id="@+id/row_order_opt"
+        android:title="Flow by row (LTR)"
```
Comment (tuomas2) [inline app/src/main/res/menu/choose_passage_book_menu.xml]: item title string need to be translatable (see example from the above item)

Comment (jmarsican) [reply app/src/main/res/menu/choose_passage_book_menu.xml]: My bad @tuomas2  I focused on other stuff and missed this. Just fixed it but strings files are conflicted and I cannot resolve them

Comment (tuomas2) [reply app/src/main/res/menu/choose_passage_book_menu.xml]: merge latest develop branch to your branch and resolve merge conflicts manually?

Comment (tuomas2) [reply app/src/main/res/menu/choose_passage_book_menu.xml]: do not edit any translations btw, just English. translations come from transifex and changes to these files will be overwritten

Comment (jmarsican) [reply app/src/main/res/menu/choose_passage_book_menu.xml]: Done!

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -851,4 +851,5 @@
     <string name="active_labels">Active labels</string>
     <string name="recent_labels">Recent labels</string>
     <string name="other_labels">Other labels</string>
+    <string name="book_menu_sort_row_opt">Flow by row (LTR)</string>
```
Comment (tuomas2) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="book_menu_sort_row_opt">Order books horizontally</string>
```

**Review by tuomas2 — APPROVED** [review]

**Comment by tuomas2** [comment]
Thank you for PR!

**Comment by oxr463** [comment]
This is awesome!

**Pull request merged**